quizzes/midterm.md

```diff
@@ -90,23 +128,40 @@
 	my_hash = { "a" => 100, "b" => 200, "c" => 300 }
 	my_hash.select {|key,value| key < "c"}
 	```
+	
+	```
+	{ nil }
+	```
 
 1. Write code that iterates through the numbers array and out`puts` the `KEY` for `each` of the values in the array.
 
 	```ruby
 	numbers = ['one', 'two', 'three']
 	```
+	
+	```
+	def init
+	  puts "numbers"
+	end
+	```
 
 1. Write code that `maps` the fruits array to a new array where all the values are `reverse`d. Eg. "apples" = "selppa"
 
 	```ruby
 	fruits = [ "apples", "bananas", "carrots", "beets" ]
 	```
+	```
+	fruits.map { |fruits| fruits.reverse } 
+	```
 
 1. Which attr_xxxxx would you use in this example? (see below)
-
+```
+attr_accessor
+```
 1. What would change in the tame method? (see below)
-
+```
+I would add one more end to close out the tame method
+```
 1. Now go tame some lions, and don't get eaten! Use the class to invoke the `tame` method on an object instance. (see below)
```

Comment:
7/10. This wouldn't actually run but you got most of the way. You need to pass in arguments when you are creating the object.